### DISCUSSION TITLE
RM551E: What is 4CC fix?

### DISCUSSION THREAD

**Discussion by ghost** [iamromulan/cellular-modem-wiki#79] [Help - Quectel]
Hi @iamromulan,

I noticed there's a file on MEGA named "**RM551E-GL 4CC fix.zip**". What is this?

Does this need to be applied to **RM551EGL00AAR01A01M8G_01.001.01.001**?

**Comment by iamromulan** [comment]
Some people are having issues getting more than 3xCA on NR SA. The file fixes it most of the time.
Inside of the zip is a file (plmn2cacombos_nr.mdb) that can be placed in the EFS partition of the modem at the location
/mdb/nr/
You have to use QPST on windows to access the EFS partition (Just google QPST)
Once placed you reboot to have it take effect. (Sometimes 2 times)

If you need to delete the file you can't do it through QPST

You have to use AT commands.

Set AT+CFUN=0
then
AT+QNVFD="/mdb/nr/plmn2cacombos_nr.mdb"
AT+CFUN=1
AT+CFUN=1,1

**Comment by zacharee** [comment]
Are there details about which PLMNs are included in the file?

**Comment by iamromulan** [comment]
> Are there details about which PLMNs are included in the file?

No idea. Got it from someone else on Quectel forums. Mainly for T-Mobile.

**Comment by zacharee** [comment]
Thanks. Do you have any tips on getting the modem to show up in EFS Explorer? I tried in ECM and NDIS modes but neither shows up. I have the AT, DM, and NMEA ports installed in device manager.

**Comment by zacharee** [comment]
Nevermind, I just forgot how to use QPST (had to add the port from the Ports tab)

**Comment by iamromulan** [comment]
There's a wiki article now :)
https://github.com/iamromulan/cellular-modem-wiki/blob/main/quectel/sdxpinn/T-Mobile_RM551EGL_patches.md
